library/src/lib/time-picker/time-picker.component.ts

```diff
@@ -108,6 +128,7 @@ export class TimePickerComponent implements ControlValueAccessor, OnInit {
         return formattedTime !== undefined ? formattedTime : '';
```

Comment:
should also identify the return type in the method head (not part of the docs issue tho)